code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1769,6 +1769,190 @@
 	reagent_flags = NONE
 	wrapped = TRUE
 
+/obj/item/weapon/reagent_containers/food/snacks/cube/roach
+	name = "Roach Cube"
+	desc = "Just add Blood!"
+	reagent_flags = REFILLABLE
+	icon = 'icons/obj/roach_cubes.dmi'
+	icon_state = "roachcube"
+	bitesize = 12
+	filling_color = "#ADAC7F"
+	center_of_mass = list("x"=16, "y"=14)
+	preloaded_reagents = list("protein" = 10)
+	taste_tag = list(MEAT_FOOD,BLAND_FOOD)
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/roach/proc/Expand()
+	src.visible_message(SPAN_NOTICE("\The [src] expands!"))
+	var/turf/T = get_turf(src)
+	new/mob/living/carbon/superior_animal/roach(T)
+	qdel(src)
+	return TRUE
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/roach/on_reagent_change()
+	if(reagents.has_reagent("blood"))
+		Expand()
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/fuhrer
+	name = "Fuhrer Cube"
+	desc = "Just add Blood!"
+	reagent_flags = REFILLABLE
+	icon = 'icons/obj/roach_cubes.dmi'
+	icon_state = "fuhrercube"
+	bitesize = 12
+	filling_color = "#ADAC7F"
+	center_of_mass = list("x"=16, "y"=14)
+	preloaded_reagents = list("protein" = 10)
+	taste_tag = list(MEAT_FOOD,BLAND_FOOD)
+
+/obj/item/weapon/reagent_containers/food/snacks/cube/fuhrer/proc/Expand()
+	src.visible_message(SPAN_NOTICE("\The [src] expands!"))
+	var/turf/T = get_turf(src)
+	new/mob/living/carbon/superior_animal/roach/fuhrer(T)
```

Comment:
Why not just replace type with variable, and cgane only var

Reply:
This^

Reply:
ok, not to be a dick, but how will that imrove or effect the function of the code?

Reply:
because i playtested it and it works as intended

Reply:
![222](https://user-images.githubusercontent.com/81935588/114287891-665c6f00-9a5a-11eb-8c6d-99e2f60bee21.png)
easier to read, consistency(Monkey cubes currently use type), and it takes less time for you to write.

Reply:
TRY 2:
![41241](https://user-images.githubusercontent.com/81935588/114288167-84c36a00-9a5c-11eb-9dc3-ccb20bc8e642.png)
It's easier to read, takes less time for you, and it's consistent since monkey cubes also use type.

Reply:
haaa, ok i'm just an idiot when it comes to code, so if you want to make the changes you want, make them here and ill approve them.

Reply:
No idea how.

Reply:
oh ok, so if you want to make edits on a PR  you go to "files changed" and in the top right corner there will be a edit option, and after those edits have been made it will ask me to approve those edits, but before making those edits, i really suggest doing a playtest of them before you make any changes here, just so it doesn't accidentally break anything.

Reply:
No option to do that. It's a pretty easy change. Even a bit easier than I made it seem in the image, since monkey cubes already use "monkey_type" to differ from species. Just do /obj/item/weapon/reagent_containers/food/snacks/monkeycube/roach
put your stuff in roach, then do /roach/(roach_type) and add anything needed like icon_state, name, and monkey_type. The microwave recipes could also be improved a bit, for example you made it so the player has to add a container full of protein to actually make the cube. I'm a dumb ass myself, but it doesn't hurt to learn a bit, assuming you have time to spare, might even save you time in the future.

Reply:
ahhhhh, this all hurts my head, everything already works, there is literally no need to change anything, other then rebalancing the recipes, Mate i understand it would make the other cubes subs of the base roach cube, but i'd prefer just to keep them separate, also this is just gonna end up in PR limbo if this just keeps carrying on.

Reply:
Pretty flimsy excuse Honestly, I would recommend doing what Cocacola suggested, it's not hard.